crates/services/txpool/src/containers/dependency.rs

```diff
@@ -294,12 +294,15 @@ impl Dependency {
                                 }
                             } else {
                                 // tx output is in pool
-                                let output_tx = txs.get(utxo_id.tx_id()).unwrap();
-                                let output =
-                                    &output_tx.outputs()[utxo_id.output_index() as usize];
-                                Self::check_if_coin_input_can_spend_output(
-                                    output, input, false,
-                                )?;
+                                let output_tx = txs.get(utxo_id.tx_id());
+                                let output = output_tx.and_then(|tx| {
+                                    tx.outputs().get(utxo_id.output_index() as usize)
+                                });
+                                if let Some(output) = output {
+                                    Self::check_if_coin_input_can_spend_output(
+                                        output, input, false,
+                                    )?;
+                                }
```

Comment:
We can revert this change if we want, it is just what fuzzer found

Reply:
I don't understand this comment

Reply:
I meant: I change this code in the past, because fuzzer found how to trigger two panics. Since we plan to remove TxPoolV1 we can revert this change, or keep it because I'm too lazy to revert this file=)

Reply:
Ah okok lets keep it for prosperity